examples/md/main.ts

```diff
@@ -0,0 +1,14 @@
+import { showdown } from "https://cdn.skypack.dev/showdown@2.1.0/dist/showdown.min.js?dts";
```

Comment:
Instead of trying to bundle Showdown into the bookmarklet, load Showdown dynamically using the function imported by the suggested import statement.
```suggestion
import { dynamicRequireShowdown } from "https://etok.codes/dynamic_require/raw/main/examples/showdown/mod.ts";
```

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/91a5e4796f9ed2a3b0e2dd26d8054d62c5895197>.\